tests/unit/__init__.py

```diff
@@ -141,11 +141,7 @@ def _report(message: str) -> None:
                       f'expected=>>>{expected_value}<<<',
                       f'comparison={op(expected_value, field_getter(record, field_name))}')
 
-        assert False, f"""
-{message}:
-
-{chr(10).join(formatted_fields)}
-"""
+        raise AssertionError(f"\n{message}:\n\n{chr(10).join(formatted_fields)}\n")
```

Comment:
This is arguable, I'd prefer `assert` to stay. These files are tests, there is no risk for them to be executed with `-O1`. `assert` in tests is IMO perfectly acceptable. Raising an exception makes to code harder to read, also in this example, the formatting is harder to spot.

Also, why just two files? There are plenty of `assert` uses under `tests/unit`, why aren't those reported by ruff?

Reply:
From [flake8-bugbear](https://github.com/PyCQA/flake8-bugbear):
"B011: Do not call assert False since python -O removes these calls. Instead callers should raise AssertionError()."

Reply:
Fair enough. In addition to B011, it also violates PT015 - "Assertion always fails, replace with `pytest.fail()`".  
noqa for both?

Reply:
Hmm, `pytest.fail()` actually sounds like a more fitting step to take here, `assert False` is just a less clear way of saying "fail here and now".